core/src/main/scala/org/apache/spark/metrics/MetricsSystem.scala

```diff
@@ -205,7 +208,7 @@ private[spark] class MetricsSystem private (
           }
         } catch {
           case e: Exception =>
-            logError("Sink class " + classPath + " cannot be instantiated")
+            logError(s"Sink class $classPath cannot be instantiated")
```

Comment:
+1 for all the log msgs. One question is should some of them be at debug level?

Reply:
Switched the added info logs to debug both here and in `MetricsConfig`.

I was originally thinking info level for these, just to make it easier to validate that metrics have been initialized. I could see someone wanting to determine, after the fact, why their metrics weren't forwarded as expected. But that said, they could always do that by relaunching with a debug log level.